concepts/05 Widgets/DataGrid/40 Scrolling/Scrolling.md

```diff
@@ -141,6 +241,62 @@ The current platform determines the native scrolling settings and you cannot adj
         ],
         // ...
     })
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <DxScrolling
+                :use-native="false"
+                :scroll-by-content="true"
+                :scroll-by-thumb="true"
+                show-scrollbar="onHover" /> <!-- or "onClick" | "always" | "never" -->
```

Comment:
The 'onClick' value is not described in documentation. But the 'onScroll' value exists.